README.md

```diff
@@ -95,6 +95,9 @@ Console.WriteLine("prediction: " + prediction.Prediction);
 ```
 A cookbook that shows how to use these APIs for a variety of existing and new scenarios can be found [here](docs/code/MlNetCookBook.md).
 
+## API Documentation
+
+See the API [here](https://docs.microsoft.com/en-us/dotnet/api/?view=ml-dotnet).
```

Comment:
>here [](start = 13, length = 4)

See the  [ML.NET API Reference Documentation]

Reply:
To make @glebuk's comment into a one-click accept: 
```suggestion
See the [ML.NET API Reference Documentation](https://docs.microsoft.com/en-us/dotnet/api/?view=ml-dotnet).
```